runtime/lua/vim/lsp/util.lua

```diff
@@ -1173,6 +1175,7 @@ function M.show_document(location, offset_encoding, opts)
   end
   if offset_encoding == nil then
     vim.notify_once('show_document must be called with valid offset encoding', vim.log.levels.WARN)
+    return false
```

Comment:
See comment further above. Please don't mix behavior changes like this into this PR